app/assets/javascripts/discourse/app/raw-views/list/new-list-header-controls.js

```diff
@@ -0,0 +1,67 @@
+import EmberObject from "@ember/object";
+import discourseComputed from "discourse-common/utils/decorators";
+import I18n from "I18n";
+
+export default EmberObject.extend({
+  @discourseComputed
+  topicsActive() {
+    return this.current === "topics";
+  },
+
+  @discourseComputed
+  repliesActive() {
+    return this.current === "replies";
+  },
+
+  @discourseComputed
```

Comment:
Should we just use native JS classes here and let these be getters? This seems odd to me considering that `discourseComputed` requires the properties that it needs to computed?

Reply:
I did try to use native classes here but they didn't work. These are raw templates/views which we're going to remove in favor of regular Ember components so I don't think we should invest too much effort in trying to modernize them.

Reply:
ahhh sorry I didn't notice that this is a raw view. No worries here.